app/labor/sticky_article_collection.rb

```diff
@@ -18,16 +17,15 @@ def user_stickies
   end
 
   def suggested_stickies
-    (tag_articles + more_articles).sample(8)
+    (tag_articles.load + more_articles).sample(8)
   end
 
   def tag_articles
-    Article.tagged_with(article_tags, any: true).
+    @tag_articles ||= Article.tagged_with(article_tags, any: true).
       includes(:user).
       where("positive_reactions_count > ? OR comments_count > ?", reaction_count_num, comment_count_num).
       where(published: true).
       where.not(id: article.id, user_id: article.user_id).
-      limited_column_select.
```

Comment:
Removed `limited_column_select` cause is wasn't helpful, cause `tagged_with` still adds `articles.*` to `select` columns list.